setup.py

```diff
@@ -425,7 +477,7 @@ def check_extension_import(self, ext):
 
         # Workaround for Cygwin: Cygwin currently has fork issues when many
         # modules have been imported
-        if host_platform == 'cygwin':
+        if HOST_PLATFORM == 'cygwin':
```

Comment:
add `CYGWIN = (HOST_PLATFORM == 'cygwin')` after `MS_WINDOWS` ?

Reply:
Yeah, I hesitated to add this one. Done, I added CYGWIN.